data/json/recipes/recipe_others.json

```diff
@@ -6658,45 +6618,27 @@
   ]
 },{
   "type" : "recipe",
-  "result": "crude_brick",
```

Comment:
Yep, this is how it's done.  You cut off the supply of non-fired bricks, get a stable release, and after that, you can kill the item in experimental.